relay-common/src/constants.rs

```diff
@@ -107,6 +107,9 @@ pub enum DataCategory {
     /// Session updates. Quantity is the number of updates in the batch.
     Session = 5,
     /// A profile
+    ///
+    /// This is the category for profiles sent alongside transactions but the transaction was
+    /// dropped.
     Profile = 6,
```

Comment:
@phacops This category should be for all profiles, regardless of whether the transaction was sampled.